components/cam/src/dynamics/se/stepon.F90

```diff
@@ -457,8 +459,15 @@ subroutine stepon_run2(phys_state, phys_tend, dyn_in, dyn_out )
       call outfld('DYN_Q'     ,dyn_in%elem(ie)%state%Q(:,:,:,1)       ,npsq,ie)
       call outfld('DYN_U'     ,dyn_in%elem(ie)%state%V(:,:,1,:,tl_f)  ,npsq,ie)
       call outfld('DYN_V'     ,dyn_in%elem(ie)%state%V(:,:,2,:,tl_f)  ,npsq,ie)
-      call outfld('DYN_OMEGA' ,dyn_in%elem(ie)%derived%omega_p(:,:,:) ,npsq,ie)
       call outfld('DYN_PS'    ,dyn_in%elem(ie)%state%ps_v(:,:,tl_f)   ,npsq,ie)
+      ! Multiply omega_p by pressure t get omega for output
+      p_tmp(:,:,1) = hvcoord%hyai(1)*hvcoord%ps0 + dyn_in%elem(ie)%state%dp3d(:,:,1,tl_f)/2
+      do k = 2,nlev
```

Comment:
I am not sure this is correct. From how I read it, p at 1st mid level will depend on dp(1) and dp(2), which is not how it is usually computed. Here is the code from another place:
```
  ! compute pressure on interfaces                                                                                   
  p_i(:,:,1)=hvcoord%hyai(1)*hvcoord%ps0
  do k=1,nlev
     p_i(:,:,k+1)=p_i(:,:,k) + dp(:,:,k)
  enddo
  do k=1,nlev
     p(:,:,k) = (p_i(:,:,k+1)+p_i(:,:,k))/2
  enddo
```
Could you please use it? 

Also, I need to check whether theta-l model computes omega/p or omega, so, this correction is not needed for theta-l.